src/EoS/Nuclear/NuclearEoS.cpp

```diff
@@ -195,11 +301,12 @@ void nuc_eos_C_short( const real xrho, real *xenr, const real xye,
    *xmunu = res[3];
    *xcs2  = res[4];
 
+#endif // #elif Table_Mode == TABLE_MODE_ENGY
```

Comment:
Maybe `#if NUC_TABLE_MODE ... else ...` is better?

Reply:
Yes. That would be better.